guides/source/upgrading_ruby_on_rails.md

```diff
@@ -358,13 +358,18 @@ puts Rails.logger.broadcasts #=> [MyLogger]
 Active Record Encryption now uses SHA-256 as its hash digest algorithm. If you have data encrypted with previous Rails
 versions, there are two scenarios to consider:
 
-1. If you have `config.active_support.key_generator_hash_digest_class` configured as SHA-1 (the default
-   before Rails 7.0), you need to configure SHA-1 for Active Record Encryption too:
+1. If you have `config.active_support.key_generator_hash_digest_class` configured as SHA-1 (the default before Rails 7.0) or have not explicitly set a configuration, you need to configure SHA-1 for Active Record Encryption too:
 
     ```ruby
     config.active_record.encryption.hash_digest_class = OpenSSL::Digest::SHA1
     ```
 
+    If all of your data was encrypted non-deterministicly (the default unless `encrypts` is passed `deterministic: true`, you can instead configure SHA-256 for Active Record Encryption as in scenario 2 below and also allow columns previously encrypted with SHA-1 to be decrypted by setting:
+
+    ```
```

Comment:
We try to always specify the langauge of the code example, for improved formatting:
```suggestion
    ```ruby
```